src/hatti/ona/forms.cljs

```diff
@@ -239,13 +264,27 @@
     (or (image? field)
         (video? field)) (let [image (:download_url answer)
                               thumb (or (:small_download_url answer) image)
+                              media-upload-fail-hlp-url
+                              urls/media-upload-fail-during-submission-help
```

Comment:
We could rename this to a more descriptive name like `urls/failed-media-upload-help-url`